rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2513,7 +2732,7 @@ bool game_list_frame::SearchMatchesApp(const QString& name, const QString& seria
 		QString title_name = m_titles.value(serial, name).toLower();
 
 		// Ignore trademarks when no search results have been yielded by unmodified search
-		static const QRegularExpression s_ignored_on_fallback(reinterpret_cast<const char*>(u8"[:\\-®©™]+"));
+		static const QRegularExpression s_ignored_on_fallback(reinterpret_cast<const char*>(u8"[:\\-Â®Â©â„¢]+"));
```

Comment:
that is also something I saw. Current master branch reports  -Â®Â©â„¢]+ (you can see from github opening the file)  while here ®©™]+ is reported as current value. Maybe it is a text displayed/converted by the editor/viewer. I will check it and revert to it to ®©™]+

Reply:
I think your IDE or git has the encoding wrong. Those A with a roof are usually control characters added by utf8 encoding

Reply:
Which means it's probably set to something different than utf8

Reply:
yes I think so (an issue on encoding). I'm using VS 2022. I will check and fix that and apply all the other changes you provided. Thanks

Reply:
fixed. This file was encoded as UTF8 instead of ANSI. Don't know if it could be better to convert on ANSI on future PRs just to avoid such kind of issue